app/models/bot/alegre.rb

```diff
@@ -14,14 +14,14 @@ class Bot::Alegre < BotUser
     
     def self.save_analysis_to_similarity_index(pm_id)
       pm = ProjectMedia.find_by_id(pm_id)
-      Bot::Alegre.send_title_to_similarity_index(pm, 'analysis_title')
-      Bot::Alegre.send_description_to_similarity_index(pm, 'analysis_description')
+      Bot::Alegre.send_field_to_similarity_index(pm, 'analysis_title')
+      Bot::Alegre.send_field_to_similarity_index(pm, 'analysis_description')
     end
 
     def self.delete_analysis_from_similarity_index(pm_id)
       pm = ProjectMedia.find_by_id(pm_id)
-      Bot::Alegre.delete_field_from_text_similarity_index(pm, 'analysis_title')
-      Bot::Alegre.delete_field_from_text_similarity_index(pm, 'analysis_description')
+      Bot::Alegre.send_field_to_similarity_index(pm, 'analysis_title')
+      Bot::Alegre.send_field_to_similarity_index(pm, 'analysis_description')
```

Comment:
@DGaffney , this should be a "delete", no?

Reply:
Yep! Already wrote some code to fix that this evening but haven’t pushed yet